TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/RoutesToProfessionalStatus/EditRoute/EndDate.cshtml.cs

```diff
@@ -50,10 +50,9 @@ public async Task<IActionResult> OnPostAsync()
             linkGenerator.RouteDetail(QualificationId, JourneyInstance.InstanceId));
     }
 
-    public async Task<IActionResult> OnPostCancelAsync()
+    public IActionResult OnPostCancel()
     {
-        await JourneyInstance!.DeleteAsync();
-        return Redirect(linkGenerator.PersonQualifications(PersonId));
+        return Redirect(linkGenerator.RouteDetail(QualificationId, JourneyInstance!.InstanceId));
```

Comment:
modified so that, from each edit page, cancel goes back to the route details page, not all the way back to the quals tab